content/en/05-reference/pxl/index.md

```diff
@@ -15,23 +15,26 @@ PxL can be executed by the Pixie platform by using either the web based UI, API
 
 ## Data Types
 
-PxL has a rich type system consisting of both concrete and semantic types.
+PxL has a rich type system consisting of both concrete and semantic types. Data types are used for the following purposes:
+
+- The query planner uses types to determine which PEM agents to pull data from.
+- The UI uses types to determine how to visualize the data. For example, data of the `ST_BYTES` semantic type will be displayed with the appropriate label (`KB`, `MB`, etc).
 
 ### Concrete Types
 
-| Type     | Description                                                        |
-| -------- | ------------------------------------------------------------------ |
-| INT64    | 64-bit integer                                                     |
-| FLOAT64  | Double precision floating point                                    |
-| TIME64NS | Time represented as 64-bit integer in nanoseconds since UNIX epoch |
-| STRING   | A UTF-8 encoded string value.                                      |
-| JSON     | A JSON value represented as a string.                              |
+The Pixie execution engine supports most concrete data types, including boolean, int64, uint128, float64, string, etc.
+
+For more info, see the complete [list of concrete data types](https://github.com/pixie-io/pixie/blob/c08aaa2c53ce95ee40817acae3f662a95994f6fb/src/api/proto/vizierpb/vizierapi.proto#L29).
```

Comment:
I would keep the concrete types around but also reomve JSON; we don't show that.